### PR TITLE
dev/database create sql statement exception and add SQLiteDBExecute

### PR DESCRIPTION
**Pull request by erinmaria98**

### CODE REVIEW

File: src/main/java/org/somerville/swag/data/source/DBConnection.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package org.somerville.swag.data.source;
 
-import org.somerville.swag.data.exception.DatabaseException;
+import org.somerville.swag.data.exception.SQLConnectionException;
```
Comment (RobertRanson) [inline src/main/java/org/somerville/swag/data/source/DBConnection.java:3]: This is a lot more specific, this will help in testing!

**Review by AlanRobinson1337 — APPROVED** [review]
Good modular design of insert and select DB statements

**Review by hzl616 — APPROVED** [review]
Great work with setting up the SQL Statements on the Java side. I like the idea of logging the rows updated after a query has run. We will need to look into the prepared statements for our SQL queries security

**Review by RobertRanson — APPROVED** [review]
Nice job!

**Review by 2013884f — APPROVED** [review]
Good work.

**Review by sgoudham — APPROVED** [review]
Really liking the new CustomExceptions that have been created which are more meaningful! Also really good that you are coding to an interface instead of an implementation. Looks great!

**Pull request merged**